### PR TITLE
feat(release): Update release-notes

### PR DESCRIPTION
**Pull request by vbotnari12**
Formulation changes.

### CODE REVIEW

File: modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -3,55 +3,54 @@
...
-On the other hand, Bonita has been using another library to display process execution data for administrators at both process and case levels. This library had some limitations and was not easy to improve with new features. +
-So it was a very natural choice to eventually integrate the bpmn-visualization library to Bonita, and team up with the Process Analytics team. +
+Till Bonita 2022.2, another library has been used to display process execution data for administrators at both process and case levels. This library had some limitations and was not easy to improve with new features. +
+Hence, integrating the bpmn-visualization library to Bonita, was a very natural choice. +
 
-Some new values are available, starting with the Efficiency Edition of Bonita:
+The new values are available for Bonita Subscription Editions starting with the Efficiency.
```
Comment (pabloalonso) [inline modules/ROOT/pages/release-notes.adoc]: ... with Efficency edition.

File: modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -3,55 +3,54 @@
...
-  * The users of the Bonita Administration application will see a faster and more robust view of the process execution data (thanks to the library and new REST APIs)
-  * As the bpmn-visualization library improves, we will add more interactivity and more information to provide a better monitoring capacity
-  * Since the pages have been recreated with the UI Designer, developers can:
+  * Bonita Administration application users will get a faster and more robust view on the process execution data (thanks to the library and new REST APIs)
+  * As the bpmn-visualization library evolves, more interactivity and information for a better monitoring capacity will be provided
+  * Since the pages have been created by using Bonita UI Designer, developers can:
         
         ** Use the custom widget embedded in the new pages in their own application pages. The widget has been wrapped into a fragment that makes it very easy to use.
-        ** Offer process execution data visualization to users of their applications, by adjusting the permissions accordingly
+        ** Offer process execution data visualization to specific users by adjusting the permissions accordingly
```
Comment (pabloalonso) [inline modules/ROOT/pages/release-notes.adoc]: What they can do is to add the page directly in their app as and link them. Why to refer to permissions here?

**Review by pabloalonso — CHANGES_REQUESTED** [review]
LGTM, but review the 2 comments before merging

File: modules/ROOT/pages/release-notes.adoc [hunk 3]
```diff
@@ -3,55 +3,54 @@
 
 [NOTE]
 ====
-The 2022.2 is in development.
+The 2022.2 is in qualification.
```
Comment (benjaminParisel) [inline modules/ROOT/pages/release-notes.adoc]: In my mind, i think we don't use `qualification` but `beta`.

File: modules/ROOT/pages/release-notes.adoc (hunk 2 above)
Comment (vbotnari12) [reply modules/ROOT/pages/release-notes.adoc]: Not sure why. This line was already here. I can remove it

File: modules/ROOT/pages/release-notes.adoc [hunk 4]
```diff
@@ -3,55 +3,58 @@
...
-  * The users of the Bonita Administration application will see a faster and more robust view of the process execution data (thanks to the library and new REST APIs)
-  * As the bpmn-visualization library improves, we will add more interactivity and more information to provide a better monitoring capacity
-  * Since the pages have been recreated with the UI Designer, developers can:
+  * Bonita Administration application users will get a faster and more robust view on the process execution data (thanks to the library and new REST APIs)
+  * As the bpmn-visualization library evolves, more interactivity and information for a better monitoring capacity will be provided
+  * Since the pages have been created by using Bonita UI Designer, developers can:
         
         ** Use the custom widget embedded in the new pages in their own application pages. The widget has been wrapped into a fragment that makes it very easy to use.
-        ** Offer process execution data visualization to users of their applications, by adjusting the permissions accordingly
+        ** Offer process execution data visualization to specific users by adjusting the permissions accordingly
```
Comment (jeromecambon) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
        ** Modify the process or case visualization page, and use it in any Bonita application
```
This is what we say in the how-to page referenced below.

**Review by jeromecambon — COMMENTED** [review]
LGTM, just a comment

File: modules/ROOT/pages/release-notes.adoc (hunk 2 above)
Comment (vbotnari12) [reply modules/ROOT/pages/release-notes.adoc]: Jerome suggested a more appropriate communication.

File: modules/ROOT/pages/release-notes.adoc (hunk 3 above)
Comment (vbotnari12) [reply modules/ROOT/pages/release-notes.adoc]: your thought is accepted :) It's indeed more accurate with the version name 👍

**Pull request merged**